crates/ruff/src/rules/flake8_pytest_style/rules/parametrize.rs

```diff
@@ -636,15 +636,13 @@ pub(crate) fn parametrize(checker: &mut Checker, decorators: &[Decorator]) {
             }) = &decorator.expression
             {
                 if checker.enabled(Rule::PytestParametrizeNamesWrongType) {
-                    if let Some(names) = args.get(0) {
+                    if let [names, ..] = &args[..] {
```

Comment:
If we're in the business of minor refactors, I tend to prefer `args.as_slice()` for this. I just find it a bit clearer.